examples/database/litefs/README.md

```diff
@@ -0,0 +1,21 @@
+# LiteFS
+
+LiteFS is a distributed file system that replicates SQLite databases to other nodes at the file system level. It works by using [FUSE](https://www.kernel.org/doc/html/next/filesystems/fuse.html) to detect writes to the file system and determines how and if those should be replicated.
+
+This example will demonstrate how to run Flipt over LiteFS.
+
+## Requirements
+
+- [Docker](https://www.docker.com/)
+- [docker-compose](https://docs.docker.com/compose/install/)
+
+## Running the Example
+
+1. Run `docker-compose up` from this directory
+1. Open the Flipt UI (default: [http://localhost:8080](http://localhost:8080))
+
```

Comment:
is there any way the user could see that the data is getting replicated? like we have in the libsql example? https://github.com/flipt-io/flipt/tree/main/examples/database/libsql

i realize we are routing the writes and reads to different instances, but its not evident when you load the ui. j/w

Reply:
@markphelps Great question. I could expose both instances, but just didn't want users to accidentally try to write on the replica instance (you would get an error).

I could just make that clear in the docs though, will do that.